hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/XceiverClientGrpc.java

```diff
@@ -280,7 +280,10 @@ public ContainerCommandResponseProto sendCommand(
       // Re-interrupt the thread while catching InterruptedException
       Thread.currentThread().interrupt();
     } catch (ExecutionException e) {
-      LOG.error("Failed to execute command {}", processForDebug(request), e);
+      LOG.error("Failed to execute command {}." +
+                      "Exception Class: {}, Exception Message: {}",
+              request.getCmdType(), e.getClass().getName(), e.getMessage());
```

Comment:
How about "if isDebugEnabled, we can log debug one, otherwise error?"
in that case, debug logging also should have some details.